recipe/patches/0006-add-ldl-to-libstdc___la_LDFLAGS.patch

```diff
@@ -0,0 +1,22 @@
+From b8b0bcd9d97f2dc59c566e0a836d3dc70ddd947b Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Mon, 15 Jul 2024 19:16:05 +1100
+Subject: [PATCH 6/6] add `-ldl` to libstdc___la_LDFLAGS
+
+---
+ libstdc++-v3/src/Makefile.am | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/libstdc++-v3/src/Makefile.am b/libstdc++-v3/src/Makefile.am
+index 37ba1491d..55622ffe7 100644
+--- a/libstdc++-v3/src/Makefile.am
++++ b/libstdc++-v3/src/Makefile.am
```

Comment:
The build script doesn't run `autoreconf` or the like.
Try something like
```sh
sed -i.bak 's/^GLIBCXX_LIBS =.*/& -ldl/' libstdc++-v3/src/Makefile.in
```
or an equivalent patch, if preferred; a patch might be nicer, but IDK if it would induce more maintenance burden (depends on how frequently the context in the generated file changes).